pkg/operator/operator.go

```diff
@@ -332,6 +334,19 @@ func (optr *Operator) sync(key string) (reconcile.Result, error) {
 	return optr.syncAll(operatorConfig)
 }
 
+func getFeatureGate(listener configlistersv1.FeatureGateLister) (*configv1.FeatureGate, error) {
+	featureGate, err := listener.Get("cluster")
```

Comment:
We would normally call this var a `lister` rather than a `listener`
```suggestion
func getFeatureGate(lister configlistersv1.FeatureGateLister) (*configv1.FeatureGate, error) {
	featureGate, err := lister.Get("cluster")
```